lib/bloc/map/map_cubit.dart

```diff
@@ -275,29 +275,43 @@ class MapCubit extends Cubit<GMapState> {
                   } else {
                     markerHue = gmap.BitmapDescriptor.hueBlue;
                   }
-
+                  var haslocation = true;
+                  if (e.isEmpty ||
+                      e.last.locationMessage == null ||
+                      e.last.locationMessage?.latitude == null ||
+                      e.last.locationMessage?.longitude == null) {
+                    haslocation = false;
+                  }
```

Comment:
```suggestion
                  final haslocation = !(e.isEmpty ||
                      e.last.locationMessage == null ||
                      e.last.locationMessage?.latitude == null ||
                      e.last.locationMessage?.longitude == null);
```

A bit unnecessary variable modification and if condition here

Reply:
suggestion adopted